packages/experimental/package.json

```diff
@@ -83,7 +83,7 @@
   },
   "peerDependencies": {
     "@rest-hooks/core": "^3.0.0-0",
-    "@rest-hooks/endpoint": "^2.0.0",
+    "@rest-hooks/endpoint": "^2.2.0-beta.2",
```

Comment:
this is a breaking change (experimental package or no)

Reply:
1. This is only breaking for beta-to-beta release, which we don't guarantee compatibility.
2. This dependency was already required in a previous breaking pr, but the specification was mistakenly not updated.

So essentially this breaking change is already covered.